markets/bfp-market/contracts/storage/Margin.sol

```diff
@@ -101,13 +101,18 @@ library Margin {
         int256 amountDeltaUsd,
```

Comment:
It's been a super long time since I've touched this and probably my fault for not using descriptive names but does it make sense to call this `profitOrLossDeltaUsd`? I was scratching my head wondering why zero or more `amountDeltaUsd` meant we can clear the debt but positive is profit, zero is profit that perfectly balances the debt and negative means more debt where the calculation of profit/loss is performed by the caller.

If you don't want to rename it, just add a comment would be useful to